apps/funding-platform/src/db/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./db.adapter";
```

Comment:
slightly nitpicking so don't take this too seriously, I find the file structure convention a bit unnecessary, why not:

```
/db
  /index.ts
  /index.spec.ts
```

as we have only 2 files, we can also put them in the root folder

```
/src
  /db.ts
  /db.spec.ts
```

If you agree, I wouldn't change it now as we may add / remove more files while the PR is being reviewed